_rules/form-field-non-empty-accessible-name-e086e5.md

```diff
@@ -46,6 +46,8 @@ The list of roles in the applicability is derived by taking all the roles from [
 - have [semantic roles][] that inherit from the [abstract](https://www.w3.org/TR/wai-aria/#abstract_roles) `input` or `select` role, and
 - do not have a [required context](https://www.w3.org/TR/wai-aria/#scope) role that itself inherits from one of those roles.
 
+Note that this rule does not test certain other control-like roles such as `button` and `menuitem`, because these do not in herit from `input` or `select`. These should be tested separately.
```

Comment:
```suggestion
Note that this rule does not test other control-like roles such as `button` and `menuitem`, because these do not inherit from `input` or `select`. These should be tested separately.
```